shell/plugins/dashboard-store/mutations.js

```diff
@@ -77,7 +69,9 @@ export function load(state, { data, ctx, existing }) {
 
     if ( entry ) {
       // There's already an entry in the store, update it
-      replace(entry, data);
+      const newData = classify(ctx, data);
```

Comment:
same for this one, where did classify happen before?

Reply:
Same as previous answer